docs/en/modules/color-utils.md

```diff
@@ -0,0 +1,62 @@
+# COLOR_UTILS Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-12-30 | [Konrad Huebner](https://github.com/skycoders) | [Konrad Huebner](https://github.com/skycoders) | [color_utils.c](../../../app/modules/color_utils.c)|
+
+The color_utils module provides some basic color transformations useful for color LEDs. It is used by the WS2812_EFFECTS module.
+
+
+## color\_utils.hsv2grb()
```

Comment:
What's the purpose of the backslash here and below?

Reply:
my one editor had an issue without the backslash, but Atom doesn't - I will remove them all.